ingester/ingester.go

```diff
@@ -165,6 +174,7 @@ func New(cfg Config, chunkStore cortex_chunk.Store) (*Ingester, error) {
 			Help: "The total number of samples returned from queries.",
 		}),
 	}
+	prometheus.MustRegister(i)
```

Comment:
> I'm hitting this with ingester tests right now

Have you written some new tests?  Wondering why I didn't hit it.

> I would propose either moving this out of the constructor again or making the metrics global. Which one do you prefer?

Global please.  For consistency sake, I'd like everything to be global - I'm not concerned about the namespace pollution, and going global results in less code, which is one of the things I care most about.